include/crow/socket_adaptors.h

```diff
@@ -1,20 +1,41 @@
 #pragma once
+
+#ifdef CROW_USE_BOOST
+#include <boost/asio.hpp>
+#include <boost/asio/version.hpp>
+#ifdef CROW_ENABLE_SSL
+#include <boost/asio/ssl.hpp>
+#endif
+#if BOOST_VERSION >= 107000
+#define GET_IO_SERVICE(s) ((boost::asio::io_context&)(s).get_executor().context())
```

Comment:
The Macro for GET_IO_SERVICE is also defined in line 23-27, 
can we combine these here?

Reply:
@gittiver done
Note that it can now only be used with `namespace asio = boost::asio;`, so it might be a good idea to give it the `CROW_` prefix

Should I do it?